app/models/creates_application_from_draft.rb

```diff
@@ -0,0 +1,70 @@
+class CreatesApplicationFromDraft
```

Comment:
I've started naming classes like this a year or so ago and I like it for three reasons:

1. Classes ending on \*or or \*er (*Creator, *Finalizer, *Converter), to me, convey an overly ambitious attempt to be generic. This class has a very special purpose and doesn't aim to be generic at all.
2. I use the verb form because I want to model the _process_, as opposed to something _static_ (like a DB record)
3. I look at the name and I know exactly what it does, helping future versions of me sticking to its single responsibility :smile: 

That being said, I'm not completely against renaming it of course. I'm against the *Delegator suffix, though, and I don't have an idea for a noun-based class name that makes it equally clear what's going on in that object.